test/controllers/callback_controller_test.rb

```diff
@@ -160,11 +167,16 @@ def mock_shopify_user_omniauth
         :shopify,
         provider: :shopify,
         uid: 'shop.myshopify.com',
-        credentials: { token: '1234' },
-        extra: { associated_user: 'user_object' }
+        credentials: {token: '1234' },
```

Comment:
{ token: '1234' }

(we recently realized that rubocop is not running on this repo so there may be some style issues 🤕)